straxen/analyses/posrec_comparison.py

```diff
@@ -21,34 +20,41 @@ def load_corrected_positions(context, run_id, events,
     :param posrec_algos: list of position reconstruction algorithms to
         use (default ['mlp', 'gcn', 'cnn'])
     """
-    
+
     posrec_algos = strax.to_str_tuple(posrec_algos)
-    
+
     if cmt_version is None:
         fdc_config = None
         try:
             fdc_config = context.get_single_plugin(run_id, 'event_positions').config['fdc_map']
             cmt_version = fdc_config[1]
         except IndexError as e:
             raise ValueError(f'CMT is not set? Your fdc config is {fdc_config}') from e
-    
-    if hasattr(cmt_version, '__len__') and not isinstance(cmt_version, str) and len(cmt_version) != len(posrec_algos):
-        raise TypeError(f"cmt_version is a list but does not match the posrec_algos ({posrec_algos}) length.")
 
-    cmt_version = (cmt_version, ) * len(posrec_algos) if isinstance(cmt_version, str) else cmt_version
+    if (
+            hasattr(cmt_version, '__len__')
+            and not isinstance(cmt_version, str)
```

Comment:
Is not it easier to ask if it is a tuple or list?

Reply:
goof point!

Reply:
good point